imap_processing/ultra/l0/ultra_utils.py

```diff
@@ -12,13 +12,13 @@ class PacketProperties(NamedTuple):
     apid: list  # List of APIDs
     logical_source: list  # List of logical sources
     addition_to_logical_desc: str  # Description of the logical source
-    width: int  # Width of binary data
-    block: int  # Number of values in each block.
+    width: Union[int, None]  # Width of binary data
```

Comment:
```suggestion
    width: Union[int, None]  # Width of binary data (may be None)
```

For these ones with possible `None` types, might be good just to make a note of that. An unsuspecting reader of this might be confused what this `Union[int, None]` means.